examples/java/core/imports/src/main/java/com/microsoft/bond/examples/Imports.java

```diff
@@ -0,0 +1,41 @@
+package com.microsoft.bond.examples;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import com.microsoft.bond.Deserializer;
+import com.microsoft.bond.Serializer;
+
+import com.microsoft.bond.protocol.CompactBinaryReader;
+import com.microsoft.bond.protocol.CompactBinaryWriter;
+
+import com.microsoft.bond.examples.imports.Message;
+import com.microsoft.bond.examples.imports.common.Priority;
+
+public class Imports {
+
+    public static void main(final String[] args) throws IOException {
+
```

Comment:
Minor: is this blank line after `main` before content intentional? Not sure what our style is for Java, having only done minimal work in it.